### ISSUE TITLE
[BUG] Bug An Error Ocurred

### ISSUE THREAD

**Issue by Alexmdz77** [SushiBtw/discord-music-player#153]
**Bug Description**
Since yesterday I get only errors: `An Error Ocurred: !p link` with any link or youtube search. This was not happening before and there were no changes on my side or on my hosting provider's side

**Reproduction Steps**
1. just !play any songs

**Versioning:**
 - OS: Debian 10
 - NPM Version 7.18.1
- Module Version 7.4.1

**Comment by SpeckyYT** [comment]
I'm unable to reproduce.
What is/are the exact error(s) you're getting?

**Comment by Alexmdz77** [comment]
```
client.player.on('error', (error) => {
			console.log(error);
			switch (error) {
				case 'QueueIsNull':
					console.log(`There is no music playing right now.`);
					break;
				case 'MessageTypeInvalid':
					console.log(`The Message object was not provided.`);
					break;
				case 'VideoUnavailable':
					console.log(`Something went wrong while playing the current song, skipping...`);
					break;
				default:
					console.log(`**An Error Ocurred:** ${error}`);
					break;
			}
```
I had added this a while ago and now I get the default error: `**An Error Ocurred:** !play https://youtu.be/84dJQ0Bx6t4` for example, every time I use !play or !playlist command

**Comment by Alexmdz77** [comment]
Ok I added all the errors to the event as in the documentation
Now i have this errors : 
![image](https://user-images.githubusercontent.com/32090268/122960450-b40f2500-d383-11eb-86c1-c8845031a02b.png)

**Comment by SpeckyYT** [comment]
Uhhhhhhhhhhh, how's the play command? It should not throw such error in theory.

**Comment by SpeckyYT** [comment]
Join the support server for this module: https://ssumbot.xyz/discord

**Comment by SushiBtw** [comment]
This issue was fixed in [v7.2.0](https://www.npmjs.com/package/discord-music-player/v/7.2.0) update that is live on NPM. Tell me if the issue was fixed 🎉
